### PR TITLE
APP_ENV environment variable consolidation

### LINKED ISSUE #5071: Consolidate APP_ENV environment variables
As part of changes in #5044, an additional environment variable was added - `SIMORGH_APP_ENV`  - because the `APP_ENV` variable is only available server-side. 

All references of `APP_ENV` should be removed & replaced with `SIMORGH_APP_ENV` for consistency, and to ensure that environment variables are available on server and client.

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves #5071

**Overall change:** 
The APP_ENV environment variable is only available server-side.  Any environment variable with the `SIMORGH_` prefix is available both on the server and the client, therefore to avoid duplication and confusion it makes sense to only have a single variable.

**Code changes:**
- Replace all references of `APP_ENV` with `SIMORGH_APP_ENV`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by karinathomasbbc** [comment]
It is worth waiting until #5044 has been merged, as this includes all of those changes too.

**Review by ryanmccombe — APPROVED** [review]

**Review by rebeccamcginn — APPROVED** [review]

**Pull request merged**
